src/main/java/org/apache/flink/benchmark/full/SerializationFrameworkAllBenchmarks.java

```diff
@@ -85,4 +90,76 @@ public void serializerAvroReflect(FlinkEnvironmentContext context) throws Except
 
 		env.execute();
 	}
+
+	@Benchmark
+	@OperationsPerInvocation(value = SerializationFrameworkMiniBenchmarks.RECORDS_PER_INVOCATION)
+	public void serializerKryoThriftWithoutRegistration(FlinkEnvironmentContext context) throws Exception {
```

Comment:
this name is a bit lengthy

Reply:
yes, unfortunately...however, it matches the naming scheme of the other benchmarks. I'm not sure we should change that without changing the other ones as well, e.g. `NoReg` as the suffix. Since all `...WithoutRegistration` benchmarks are in the `SerializationFrameworkAllBenchmarks` class, this shouldn't be the problem, I guess. Tell me what you think...

Reply:
Or...since there is virtually no difference between the additional registration call and omitting it, we could completely remove this difference here and just use one test without the extra registration